src/domain/session/room/UnknownRoomViewModel.js

```diff
@@ -15,17 +15,25 @@ limitations under the License.
 */
 
 import {ViewModel} from "../../ViewModel";
+import {TimelineViewModel} from "./timeline/TimelineViewModel";
+import {tileClassForEntry as defaultTileClassForEntry} from "./timeline/tiles/index";
+import {getAvatarHttpUrl} from "../../avatar";
 
 export class UnknownRoomViewModel extends ViewModel {
     constructor(options) {
         super(options);
-        const {roomIdOrAlias, session} = options;
+        const {roomIdOrAlias, session, peekable} = options;
```

Comment:
This is a generic comment for the whole PR and not about this specific line.

Looking at the spec for [Room History Visibility](https://spec.matrix.org/v1.5/client-server-api/#room-history-visibility), the term defined there is `world_readable`, instead of `peekable`:

> `world_readable` - All events while this is the `m.room.history_visibility` value may be shared by any participating homeserver with anyone, regardless of whether they have ever joined the room.

I think it's generally best to use in the code the same terms that users use, since users could already be familiar with the spec-defined term. Doing this makes it easier to understand the code, by removing one layer of indirection.

So my suggestion would be to use the term `wordReadable` instead of `peekable` in the entirety of the codebase. Some examples:

```js
get kind() {
    return this._worldReadable ? "worldReadableRoom" : "unknown";
}

this._session.loadWorldReadableRoom(this.roomIdOrAlias);
```

Reply:
Good idea! I updated that throughout the PR ✅